Specify the default batch mode for each backend type.

MohammadAlavi1986 opened the pull request:
### Specify the default batch mode for each backend type.

The default batch mode for `webhook` backend is `batch` and `blocking` for `log` backend.  However, the current documentation gives the impression that `batch` is the default mode for both backend types.

sftim commented on content/en/docs/tasks/debug/debug-cluster/audit.md:
How about using [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) to provide details for webhook mode and for log mode, in parallel?

MohammadAlavi1986 replied:
sounds good

MohammadAlavi1986 commented on content/en/docs/tasks/debug/debug-cluster/audit.md:
```diff
@@ -201,8 +201,8 @@ name. By default, batching is enabled in `webhook` and disabled in `log`. Simila
-  - `batch` - buffer events and asynchronously process them in batches. This is the default.
-  - `blocking` - block API server responses on processing each individual event.
+  - `batch` - buffer events and asynchronously process them in batches. This is the default mode for `webhook` backend.
+  - `blocking` - block API server responses on processing each individual event. This is the default mode for `log` backend.
```
> To get the same flag for log backend, replace webhook with log in the flag name
I think the documentation focuses only on the `webhook` backend flags for brevity, as the `log` backend flags are identical.

sftim commented:
@MohammadAlavi1986 would you have capacity / be willing to revise this based on feedback so far?

MohammadAlavi1986 commented:
> @MohammadAlavi1986 would you have capacity / be willing to revise this based on feedback so far?
I added tabs to list flags specific to each backend separately. I also added `--audit-*-batch-throttle-enable` flag to the list of flags and specified the default value for each flag.

MohammadAlavi1986 commented:
> @MohammadAlavi1986 would you have capacity / be willing to revise this based on feedback so far?

I added tabs to list flags specific to each backend separately. I also added --audit-*-batch-throttle-enable flag to the list of flags and specified the default value for each flag.

sftim reviewed (commented):
Thanks

/lgtm

lmktfy commented:
/lgtm

reylejano commented:
/approve
